sgl-kernel/benchmark/bench_moe_align_block_size.py

```diff
@@ -360,15 +381,15 @@ def benchmark(num_tokens, num_experts, topk, provider):
     parser.add_argument(
         "--num_experts",
         type=int,
-        default=256,
+        default=8,
```

Comment:
nit: curious why do we change the defaults (seems 256 experts w/ 8 topk is the value for deepseek)

Reply:
I'll revert it.